package.json

```diff
@@ -4,7 +4,7 @@
     "test": "npm run with-server 'npm-run-all test:*'",
     "test:rust": "cargo test",
     "test:python": "cd python-sdk && pytest",
-    "test:ruby": "cd ruby-sdk && bundle exec rake test",
+    "test:ruby": "cd ruby-sdk && bundle exec rake build && bundle exec rspec",
```

Comment:
This is the fix for CI: `bundle exec rake test` does not run rspec

Reply:
Good find - thank you for adding this.